balrogscript/docker.d/worker.yml

```diff
@@ -69,3 +69,21 @@ server_config:
       - 'esr'
       - 'esr-localtest'
       - 'esr-cdntest'
+  staging:
+    api_root: { "$eval": "STAGE_API_ROOT" }
+    auth0_domain: { "$eval": "AUTH0_DOMAIN"}
+    auth0_client_id: { "$eval": "AUTH0_CLIENT_ID" }
+    auth0_client_secret: { "$eval": "AUTH0_CLIENT_SECRET" }
+    auth0_audience: { "$eval": "AUTH0_AUDIENCE" }
+    allowed_channels:
```

Comment:
For staging, I wonder if there's a way to say "all channels are allowed".

Not necessary. But for system addons we may at least want to add `*-sysaddon` since that's how we named our test channels there https://github.com/mozilla-releng/systemaddons-wip/tree/main/existing/SystemAddons/rules .

Reply:
Should we add `*-sysaddon` to the other servers? This might be relevant to the balrog rules

Reply:
I think so.

`*-sysaddon` only really applies to the xpi balrog scriptworkers, so this is also a place where we could have a different set of allowed channels depending on the trust domain. If that's easy to do, let's do that; if that's trickier to do, it's probably not a huge worry if we're able to push to the other products' test channels.